### ISSUE TITLE
Updating via v0.11.1 binaries shows "version": "0.11.0.1"

### ISSUE THREAD

**Issue by BTCBellyButton** [ElementsProject/lightning#5273]
### Issue and Steps to Reproduce
After installing from the binaries (wget https://github.com/ElementsProject/lightning/releases/download/v0.11.1/clightning-v0.11.1-Ubuntu-20.04.tar.xz)

The version shown is still 0.11.0.1:

"version": "0.11.0.1",
   "blockheight": 736987,
   "network": "bitcoin",

**Comment by vincenzopalazzo** [comment]
I think this is just a typo! there is nothing to worry about!

**Comment by BTCBellyButton** [comment]
> I think this is just a typo! there is nothing to worry about!

Of course. It's just confusing when checking if the update has been done correctly. It makes you waste some time re-checking what you've done.   ;-)

**Comment by vincenzopalazzo** [comment]
>Of course. It's just confusing when checking if the update has been done correctly. It makes you waste some time re-checking what you've done. ;-)

Agree!

**Comment by rustyrussell** [comment]
AI cleanup of old issues.  If wrong, please open a new issue.


Triage decision summary:
Version string mismatch in v0.11.1 binaries showing 0.11.0.1 instead. Likely a historical build/packaging issue from May 2022 that would have been resolved in subsequent releases.
